doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -263,13 +263,17 @@ The diagnostic logs snippet enables the set of configurations needed for full Ma
 The configuration set consist of devicetree overlays for each supported target board, and a config file that enables all diagnostic logs features by default.
 The devicetree overlays add new RAM partitions which are configured as retained to keep the log data persistent and survive the device reboot.
 They also reduce the SRAM size according to the size of the retained partition.
+The parition sizes are configured using example values and may not be sufficient for your use cases.
+To change the partition sizes, for example increase them to be able to store more logs, you need to change the configuration in the devicetree overlay.
 The snippet sets the following kconfig options:
 
   * :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS` to ``y``.
   * :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_CRASH_LOGS` to ``y``.
   * :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REMOVE_CRASH_AFTER_READ` to ``y``.
   * :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_END_USER_LOGS` to ``y``.
   * :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_NETWORK_LOGS` to ``y``.
+  * :kconfig:option:`CONFIG_LOG_MODE_DEFERRED` to ``y``.
```

Comment:
Should we have more comments on that, or do you think it is redundant?

Reply:
I can add additional explanation, similarly to the one that is in the .conf files's comment.

Reply:
Done.